app/controllers/wizard/steps/planned_processing_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Wizard
+  module Steps
+    class PlannedProcessingController < BaseController
+      def show
+        @step = Wizard::Steps::PlannedProcessing.new(user_session)
+      end
+
+      def create
+        @step = Wizard::Steps::PlannedProcessing.new(user_session, permitted_params)
+
+        if @step.valid?
+          @step.save
+
+          redirect_to @step.next_step_path(
+            service_choice: params[:service_choice],
+            commodity_code: params[:commodity_code],
+          )
+        else
+          render 'show'
+        end
```

Comment:
Ah if you pulled the latest master this has been replaced with `validate(@step)` so you can use this instead of 
```ruby
if @step.valid?
          @step.save

          redirect_to @step.next_step_path(
            service_choice: params[:service_choice],
            commodity_code: params[:commodity_code],
          )
        else
          render 'show'
        end

```